### DISCUSSION TITLE
How to send a POST http with request body

### DISCUSSION THREAD

**Discussion by lvbaiquan12** [vectordotdev/vector#15629] [Q&A] [answered]
With the Sources type "http_client", how to send a POST http with request body. I know how to send a "GET" request, but I do not know which filed to construct POST request body.  I am trying to send a POST request with the request body to obtain the returned json results.

**Comment by spencergilbert** [comment, marked as answer]
👋 hi @lvbaiquan12,

That isn't currently supported but there's an open issue to add support https://github.com/vectordotdev/vector/issues/14758, give it a 👍 to help us prioritize the work - or if you're interesting in contributing we're happy to help get a pull request merged!

**Reply by lvbaiquan12** [reply]
Thank you for your answer.I have given it a 👍.
